scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -0,0 +1,284 @@
+"""Module to generate a column map to their corresponding statistical variable defintions."""
+import os
+import sys
+import logging
+import json
+import re
+
+import pandas as pd
+
+#TODO:Remove this import and make it from ~/data/utils/ directory when it is available
+os.system(
+    'curl https://raw.githubusercontent.com/Abilityguy/data/dcid_gen/util/statvar_dcid_generator.py -LO statvar_dcid_generator.py'
+)
+from statvar_dcid_generator import get_stat_var_dcid
+
+# intitalize the logger
+logging.basicConfig(
+    filename='column_map_generatetion_task.log',
+    format="{asctime} {processName:<12} {message} ({filename}:{lineno})",
+    style="{",
+    filemode='w')
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+# regex for the definition of QuantityRanges
+# From: utils/statvar_dcid_generator.py
+_QUANTITY_RANGE_REGEX = (r"\[(?P<lower_limit>-|-?\d+(\.\d+)?) "
+                         r"(?P<upper_limit>-|-?\d+(\.\d+)?) "
+                         r"(?P<quantity>[A-Za-z]+)\]")
+
+# list of properties that every statistical variable should contain
+_MANDATORY_PROPS = ['measuredProperty', 'populationType', 'statType']
+
+# common properties and their associated populationTypes
+# TODO: Establish if this map generalizes across the different subject tables
+_DISAMBIGUATING_POP_TYPE = {'income': 'Household', 'earnings': 'Person'}
```

Comment:
I don't think this is true generally. There is income for Person too.  I was suggesting adding this to the config:

```
		"earnings": {
			"$1 to $4999 or loss": "[- 4999 USDollar]",
			"$5000 to $14999": "[5000 14999 USDollar]",
			"$15000 to $24999": "[15000 24999 USDollar]",
			"$25000 to $34999": "[25000 34999 USDollar]",
			"$35000 to $49999": "[35000 49999 USDollar]",
			"$50000 to $74999": "[50000 74999 USDollar]",
			"$75000 or more": "[75000 - USDollar]",
                         "_DISAMBIGUATING_POP_TYPE": "Person"
		},
		"income": {
			"Under $25000": "[- 25000 USDollar]",
			"$25000 to $49999": "[25000 49999 USDollar]",
			"$50000 to $74999": "[50000 74999 USDollar]",
			"$75000 to $99999": "[75000 99999 USDollar]",
			"$100000 and over": "[100000 - USDollar]",
                         "_DISAMBIGUATING_POP_TYPE": "Household"
		},
```

so that it is config specific